lib/Backend/GlobOpt.cpp

```diff
@@ -3033,13 +3038,13 @@ GlobOpt::OptDst(
         else if (dstVal)
         {
             opnd->SetValueType(dstVal->GetValueInfo()->Type());
-
+#if 0
```

Comment:
can we really remove this?

Reply:
Louis wasn't sure what the intent of this code is so he asked me to just #if 0 that block for now. So far there doesn't seem to be any problems with not having it though.

Reply:
have you checked perf of octane after your change?

---
In reply to: [310843120](https://github.com/Microsoft/ChakraCore/pull/6226#discussion_r310843120) [](ancestors = 310843120)

Reply:
I have not, but the `#if 0` block only runs for generator functions (last condition) so we should be fine regardless  right?

Reply:
no, because `DoGlobOptsForGeneratorFunc` used to return true for all non-generator functions

---
In reply to: [311200623](https://github.com/Microsoft/ChakraCore/pull/6226#discussion_r311200623) [](ancestors = 311200623)

Reply:
Ahh right. I'll fix this. :)